TROUBLESHOOTING.md

```diff
@@ -0,0 +1,52 @@
+# Troubleshooting
+
+## "ALPN is not configured properly"
+
+This exception usually indicates that Netty isn't set up properly.
+
+If using gRPC version 1.19.0 or newer, consider using `io.grpc:grpc-netty-shaded`
+dependency so that it does not conflict with other packages that also require Netty.
+
+If your gRPC dependency is older than 1.19.0, consider updating.
+As of 0.35.0, packages in this `google-cloud-java` already declares dependency on `grpc-netty-shaded`.
+
+The versions of `gprc-netty-shaded` must match the version of other gRPC dependencies.
+Packages in this `google-cloud-java` already does this for you.
+However, if you need to declare dependency on gRPC yourself, we recommend
```

Comment:
s/recommend/recommend doing it like this:/

Reply:
Done